### PR TITLE
feat: add find command

### PR DESCRIPTION
**Pull request by dead-horse**

### CODE REVIEW

**Comment by dead-horse** [comment]
@popomore CI 过了

File: lib/command/find.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const path = require('path');
+const chalk = require('chalk');
+const inquirer = require('inquirer');
+const clipboardy = require('clipboardy');
+const BaseCommand = require('../base_command');
+
+class AddCommand extends BaseCommand {
```
Comment (popomore) [inline lib/command/find.js]: FindCommand

**Comment by popomore** [comment]
I prefer to add hook for all command, but I don't is it useful.

so we can add `prefind` and `postfind` hook.

File: lib/command/find.js (hunk 1 above)
Comment (dead-horse) [inline lib/command/find.js]: fixed

**Comment by popomore** [comment]
I will add hook in another PR

**Pull request merged**